src/main/java/com/iota/iri/network/TCPNeighbor.java

```diff
@@ -93,7 +93,7 @@ public void send(DatagramPacket packet) {
         synchronized (sendQueue) {
             if (sendQueue.remainingCapacity() == 0) {
                 sendQueue.poll();
-                log.info("Sendqueue full...dropped 1 tx");
+                log.debug("Sendqueue full...dropped 1 tx");
```

Comment:
I found this info statement very useful while trying to peer nodes and noticing right away that there is a problem.
I agree it is very annoying the way it can flood the console.

Maybe use the `IntervalLogger`?

Reply:
We already have the connection message for this case. In case a node drops the connection or doesnt accept packets (like with NELSON) you will not see anything in your cli.